cms/templates/base.html

```diff
@@ -155,7 +156,8 @@
           num: "${context_course.location.course | n, js_escaped_string}",
           display_course_number: "${context_course.display_coursenumber | n, js_escaped_string}",
           revision: "${context_course.location.branch | n, js_escaped_string}",
-          self_paced: ${ context_course.self_paced | n, dump_js_escaped_json }
+          self_paced: ${ context_course.self_paced | n, dump_js_escaped_json },
+          is_custom_pls_active: ${custom_pls_is_active(context_course.id) | n, dump_js_escaped_json}
```

Comment:
Nit: keep consistent naming. You have `is_custom_pls_active` and `custom_pls_is_active`. This might become a moot point if you end up removing the utility function, but otherwise, I'd advocate for keeping it consistent.